src/_stylesheets/_inform-inspire.scss

```diff
@@ -1,80 +1,135 @@
+  }
+}
+
+// Overriding .govuk-list's spacing so that it doesn't add extra space at the
+// bottom of the block and creates space between the content element above
+.app-inform-inspire__list {
+  @include govuk-responsive-margin(4, $direction: 'top');
+  margin-bottom: 0;
+}
+
+// Scale bar elements on the homepage
+.app-inform-inspire__bar {
+  @include govuk-font($size: 19, $weight: bold);
+  @include govuk-responsive-padding(4);
+  box-sizing: border-box;
+  width: 100%;
+  margin-bottom: govuk-spacing(1);
+  color: govuk-colour('white');
+
+  &:nth-of-type(1) {
+    width: calc(100% / 3);
+    background-color: govuk-colour('blue');
+  }
+
+  &:nth-of-type(2) {
+    width: calc((100% / 3) * 2);
+    background-color: govuk-colour('green');
+  }
+
+  &:nth-of-type(3) {
+    width: 100%;
+    background-color: govuk-colour('pink');
+  }
+
+  &:last-of-type {
+    margin-bottom: 0;
+  }
+}
+
+// Inform <----> Inspire axis visual element
+.app-inform-inspire__axis {
+  display: flex;
+  position: relative;
+  margin-bottom: govuk-spacing(5);
+  color: govuk-colour('black');
+  justify-content: space-between;
 
   &::before {
     content: '';
     box-sizing: border-box;
     display: block;
+    position: absolute;
+    bottom: $bottom-position;
     width: $arrow-width;
     height: $arrow-width;
-    margin-left: -$padding-left;
-    transform: rotate(45deg) translateX(-$border-width) translateY(-1px);
+    transform: rotate(-45deg);
     border-top: solid $border-width;
     border-left: solid $border-width;
-    transform-origin: top left;
   }
 
   &::after {
     content: '';
     box-sizing: border-box;
     display: block;
+    position: absolute;
+    right: 0;
+    bottom: $bottom-position;
     width: $arrow-width;
     height: $arrow-width;
-    margin-left: -$padding-left;
-    transform: rotate(-45deg) translateX(-$border-width) translateY(1px);
-    border-bottom: solid $border-width;
+    transform: rotate(-225deg);
+    border-top: solid $border-width;
     border-left: solid $border-width;
-    transform-origin: bottom left;
-  }
-
-  figure {
-    margin-inline: 0;
-  }
-
-  figure div {
-    width: 60%;
-    margin: 0 auto;
-  }
-
-  figcaption {
-    margin-top: govuk-spacing(1);
-
-    p {
-      margin-bottom: 0;
-    }
-  }
-
-  > * {
-    margin-block: 0;
-  }
-
-  > * + * {
-    margin-top: $padding-left;
   }
+}
 
-  .app-inform-inspire__inform {
-    margin-top: -16px;
+.app-inform-inspire__axis-label {
+  @include govuk-font($size: 19, $weight: bold);
+  width: 42%;
+  padding-bottom: govuk-spacing(5);
+  border-bottom: $border-width solid;
+  color: $app-blue-shade-50;
 
-    + * {
-      margin-top: 0.75 * $gap;
-    }
+  &:last-of-type {
+    text-align: right;
   }
 
-  .app-inform-inspire__inspire {
-    margin-top: 0.5 * $gap;
-    margin-bottom: -16px;
+  @include govuk-media-query($from: desktop) {
+    width: 45%;
   }
+}
 
-  img {
-    width: 100%;
-    height: auto;
-  }
+.app-inform-inspire__axis-dot {
```

Comment:
This is also not visible in forced colours mode, though arguably this is less important given it's a purely decorative element.